src/docs/release-notes/R17/R17.mdx

```diff
@@ -0,0 +1,20 @@
+### Stickersheet Components
+
+-   Added PX Blue components (EmptyState, ChannelValue, Heroes, ListItemTag, InfoListItem, NavRail)
+-   Added more Material UI components (Bottom Sheets, Chips, Snackbars, Cards, Sliders)
+
+### Design Patterns
+
+-   Added more images and gifs to showcase each design pattern.
+-   Reorganized each design pattern page to have a behaviors, anatomy, specifications, and developer section.
+-   Completed migration of [Angular](https://github.com/pxblue/angular-design-patterns), [Ionic](https://github.com/pxblue/ionic-design-patterns), and [React Native](https://github.com/pxblue/react-native-design-patterns) design patterns into a mono-repo.
+
+### Component Libraries
+
+-   Added [Spacer](https://pxblue-components.github.io/react-native/?path=/info/components-documentation--spacer) and [List Item Tag](https://pxblue-components.github.io/react-native/?path=/info/components-documentation--list-item-tag) components to React Native
+-   Added [Dropdown Toolbar](https://pxblue-components.github.io/react/?path=/info/components-dropdown-toolbar--get-read-me-story) component to React
+-   Added [Dropdown Toolbar](https://pxblue-components.github.io/angular/?path=/info/components-dropdown-toolbar--readme) component and Nav-Rail component to Angular
```

Comment:
Follow the same (component libraries) format we used for R16

Reply:
I updated the format to match what was done for R16, also added a note about progress icon labels and the new UPS icon.

Reply:
Forgot that we have a storybook addon for RTL support as well.